core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -1,137 +1,943 @@
+          arrow_forward_ios
+        </span>
+      </div>
+    </div>
+    <div  *ngIf="!isWindowNarrow" class="oppia-teach-section-five-right">
+      <picture class="teach-student-image">
+        <source type="image/webp" [srcset]="getStaticImageUrl(testimonials[displayedTestimonialId].imageUrlWebp)" >
+        <source type="image/png" [srcset]="getStaticImageUrl(testimonials[displayedTestimonialId].imageUrl)" >
+        <img [src]="getStaticImageUrl(testimonials[displayedTestimonialId].imageUrl)" [ngClass]="{'image-with-border': testimonials[displayedTestimonialId].borderPresent}" alt="">
+      </picture>
+    </div>
+
+    <div *ngIf="isWindowNarrow">
+      <div class="oppia-teach-section-five-image">
+        <picture class="teach-student-image">
+          <source type="image/webp" [srcset]="getStaticImageUrl(testimonials[displayedTestimonialId].imageUrlWebp)" >
+          <source type="image/png" [srcset]="getStaticImageUrl(testimonials[displayedTestimonialId].imageUrl)" >
+          <img [src]="getStaticImageUrl(testimonials[displayedTestimonialId].imageUrl)" [ngClass]="{'image-with-border': testimonials[displayedTestimonialId].borderPresent}" alt="">
+        </picture>
+      </div>
+      <div class="oppia-teach-section-five-left mobile">
+        <h2 class="mobile-h2" [innerHTML]="testimonials[displayedTestimonialId].quote | translate"></h2>
+        <h3>-
+          <h3 [innerHTML]="testimonials[displayedTestimonialId].studentDetails | translate"></h3>
+        </h3>
+        <div class="teach-arrows-container">
+          <span (click)="decrementDisplayedTestimonialId()" class="material-icons">
+            arrow_back_ios
+          </span>
+          <span (click)="incrementDisplayedTestimonialId()" class="material-icons">
+            arrow_forward_ios
+          </span>
         </div>
       </div>
     </div>
   </div>
-  <div class="oppia-footer-padding-below-banner">
+
+  <div class="oppia-teach-section-six">
+    <div *ngIf="!isWindowNarrow">
+      <h2 [innerHTML]="'I18N_TEACH_PAGE_SIX_TITLE' | translate"></h2>
+      <!--<div class="oppia-teach-button-container-button-one">
+        <form action="/teach">
+          <button type="submit" (click)="onClickGuideParentsButton()"
+                  class="btn oppia-teach-bottom-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_GUIDE_FOR_TEACHERS' | translate">
+          </button>
+        </form>
+      </div>
+      <div class="oppia-teach-button-container-button-one">
+        <form action="/teach">
+          <button type="submit" (click)="onClickTipforParentsButton()"
+                  class="btn oppia-teach-bottom-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_TIPS_FOR_PARENTS' | translate">
+          </button>
+        </form>
+      </div> -->
```

Comment:
I haven't seen commented code anywhere else in the codebase.
Are we okay with this here?
@Rijuta-s @kevintab95 @aks681

Reply:
It was discussed in the PR thread and an explanation for the same is already added above the code. These buttons are temporarily removed because the pages were not ready. As soon as they are ready we will get this uncommented.

Reply:
Okay!